source/photo.html

```diff
@@ -8,27 +8,27 @@
   </head>
   <body>
     <header class="header">
-
-      <a href="index.html" class="header__logo">
-        <img src="https://via.placeholder.com/150x150" width="20" height="20" alt="Логотип сайта">
-      </a>
-
-      <button class="header__main-nav__toggle" type="button">
-        <span class="visually-hidden">Меню</span>
-      </button>
-
-      <nav class="header__main-nav main-nav">
+      <div class="header__logo-toggle-wrapper">
+        <a href="index.html" class="header__logo">
+          <img src="https://via.placeholder.com/150x150" width="20" height="20" alt="Логотип сайта">
+        </a>
+
+        <button class="header__main-nav__toggle" type="button">
```

Comment:
```suggestion
        <button class="header__main-nav-toggle" type="button">
```

Некорректный класс.
`header__` - Блок header
`header__main-nav` - элемент блока header - main-nav
`__toggle` - а это что? это же не элемент элемента main-nav) это элемент блока `header`, так что через `__` его написать не получится